modules/accredited_representative_portal/db/migrate/20250313190720_create_ar_icn_temporary_identifiers.rb

```diff
@@ -0,0 +1,9 @@
+class CreateArIcnTemporaryIdentifiers < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_icn_temporary_identifiers, id: :uuid do |t|
```

Comment:
I was also thinking through whether the word `claimant` should show up in the table name or a column. This was also interesting. I think we could include it in the table name to codify the fact that this is not for representative user accounts (for which it is okay for us to use the user account id in our app), but is for other parties, which are potentially only claimants by definition. But I think we can't know that last bit, so leaving this generically about ICNs makes sense to me.